lib/taurus/qt/qtgui/base/taurusbase.py

```diff
@@ -1342,21 +1366,52 @@ def __init__(self, name='', parent=None, designMode=False, **kwargs):
         self.call__init__(TaurusBaseComponent, name,
                           parent=parent, designMode=designMode)
         self._setText = self._findSetTextMethod()
+        self._formatters = None
 
     def showFormatterDlg(self):
         """
         showFormatterDlg show a dialog to get the formatter from the user.
         :return: formatter: python fromat string or formatter callable
         (in string version) or None
         """
-        current_format = self.getFormat()
+        if self._formatters is None:
+            # initialize cache
+            self._formatters = TaurusBaseWidget.get_registered_formatters()
+
+        # Find index of the current formatter
+        try:
+            ind = self._formatters.values().index(self.FORMAT)
+        except ValueError:
+            ind = 0
+
+        uri = "http://taurus-scada.org/devel/api/taurus/qt/qtgui/base/_TaurusBaseComponent.html"
+        help = "<a href=\"{}\"> Find more info in doc</a>".format(uri)
```

Comment:
Not crytical, but the link does not launch a browser in my system when clicked (but it allows to right-click and copy the URL which , when pasted on a browser works as expected)